src/Tizen.NUI/src/internal/Interop/Interop.TextEditor.cs

```diff
@@ -215,6 +215,15 @@ internal static partial class TextEditor
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_TextEditor_Property_PLACEHOLDER_get")]
             public static extern int TextEditor_Property_PLACEHOLDER_get();
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_SelectWholeText")]
```

Comment:
Hello, 
it seems that there must be a dali-csharp-binder's patch paired with this. 
could you inform the patch link here?

Reply:
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-csharp-binder/+/245872